bin/run_bam_qc.py

```diff
@@ -93,10 +103,22 @@ def main():
     parser.add_argument('-t', '--target', metavar='PATH',
                         help='Path to target BED file, containing targets to calculate coverage '+\
                         'against. Optional; if given, must be sorted in same order as BAM file.')
-    parser.add_argument('-T', '--temp-dir', metavar='PATH', help='Directory for temporary output files; optional, defaults to /tmp')
-    parser.add_argument('-v', '--verbose', action='store_true', help='Print additional messages to STDERR')
+    parser.add_argument('-T', '--temp-dir', metavar='PATH', help='Directory for temporary output files; optional, defaults to %s (the current system tempdir).' % tempfile.gettempdir())
+    parser.add_argument('-v', '--version', action='store_true',
+                        help='Print the version number of bam-qc-metrics and exit')
```

Comment:
argparse has a buildin way to deal with versions documented here https://docs.python.org/dev/library/argparse.html#action and explained better here https://stackoverflow.com/questions/15405636/pythons-argparse-to-show-programs-version-with-prog-and-version-string-formatt

Reply:
I wasn't aware of that. Very useful, I'll make the change. :-)

Reply:
Using this might allow you to have `--bam` required again (I am curious if that is a bug with `argparse`)